modules/UserDashboard/components/AllClassrooms/AllClassrooms.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Button, Text } from "@mantine/core";
+import { FaPlus } from "react-icons/fa";
+import { useSelector } from "react-redux";
+
+import { TRootState } from "@/shared/redux/store";
+import { ERole } from "@/shared/typedefs";
+
+import classes from "./AllClassrooms.module.css";
+
+const AllClassrooms = () => {
+  const user = useSelector((state: TRootState) => state.authenticatedUser);
```

Comment:
```suggestion
  const user = useAppSelector(authenticatedUserSelector);
```

Should be typed. Look into the template for useAppSelector implementation.

Should also add a const for whatever you're trying to fetch from the reducer slice and import it here. authenticatedUserSelector in this case